client/homebrew/pages/newPage/newPage.jsx

```diff
@@ -55,19 +56,31 @@ const NewPage = createClass({
 
 		const brew = this.state.brew;
 
-		if(!this.props.brew.shareId && typeof window !== 'undefined') { //Load from localStorage if in client browser
+		if (!this.props.brew.shareId && typeof window !== 'undefined') {
+			// Load from localStorage if in client browser
 			const brewStorage  = localStorage.getItem(BREWKEY);
 			const styleStorage = localStorage.getItem(STYLEKEY);
-			const metaStorage = JSON.parse(localStorage.getItem(METAKEY));
-
+			const sizeStorage  = localStorage.getItem(SIZEKEY);
+			const metaStorage  = JSON.parse(localStorage.getItem(METAKEY));
+		
 			brew.text  = brewStorage  ?? brew.text;
 			brew.style = styleStorage ?? brew.style;
-			// brew.title = metaStorage?.title || this.state.brew.title;
-			// brew.description = metaStorage?.description || this.state.brew.description;
+		
+			// Check if a page size is already defined in brew.style
+			const pageSizeRegex = /\.page\s*{[^}]*\b(width|height)\b[^}]*}/;
+			if (!pageSizeRegex.test(brew.style)) {
+				// No page size defined, apply the default size if sizeStorage is 'A4', otherwise the default letter stays
+				if (sizeStorage === 'A4') {
+					brew.style = brew.style + '\n\n/* A4 Page Size */\n.page{\n    width  : 210mm;\n    height : 296.8mm;\n}\n';
+				}
+			}
```

Comment:
If this PR is only adding page size to `/new`, I would say it should specifically only add page size to EMPTY `/new` brews, so it doesn't surprise users with a sudden size change in the background if they return to a work in progress, or clone an existing brew. (i.e., imagine your personal setting is A4, but you clone a brew without a page size. The cloned brew will appear broken and it may not be obvious why the text content doesn't fit anymore).

This would also simplify this step, as we don't need to scan for an existing size. Just apply the size when a brand new brew is created.